tests/integ/__init__.py

```diff
@@ -25,5 +25,10 @@
 PYTHON_VERSION = 'py' + str(sys.version_info.major)
 REGION = boto3.session.Session().region_name
 
+# https://code.amazon.com/packages/MaeveControlPlaneService/blobs/mainline/--/src/com/amazonaws/services/ironman/utils/InstanceTypesHelper.java
```

Comment:
I guess we should not link to internal resources on github.

Reply:
right